lilysolarsystem3.rb

```diff
@@ -0,0 +1,125 @@
+#Lily Solar System: Wave III
+
+#Create planet class with specified information for all planets
+class Planet
+  attr_accessor :name, :gname, :moon, :au
+
+  def initialize(planet_name, planet_gname, planet_moon, planet_au)
+    @name = planet_name
+    @gname = planet_gname
+    @moon = planet_moon
+    @au = planet_au
+  end
+end
+
+#Create overarching solar system class with planets as the instance variable
+class SolarSystem
+  attr_accessor :planets
```

Comment:
Why make this an `attr_accessor`, instead make it an `attr_reader`.  That way you don't give users of `SolarSystem` the ability to modify the list of planets directly.  

In general don't give users any more access to attributes than you must.